omod/src/main/webapp/pages/labtrackingViewQueue.gsp

```diff
@@ -60,12 +60,6 @@ ${patient?ui.includeFragment("coreapps", "patientHeader", [ patient: patient ])
                </label>
             </div>
 
-            <div class="col-md-6 text-left">
-               <label>
-                  <input id="urgentReview-search" type="checkbox"/>
-                  ${ui.message("labtrackingapp.orderdetails.urgentreviewlabel")}
-               </label>
-            </div>
```

Comment:
Same question here...?

Reply:
Yes, this was intentional. UHM-6185 says that urgentReview filter is optional (or less important than the suspectedCancer). It is easy to add it, but first I wanted to finish the mandatory items.